xarray/core/utils.py

```diff
@@ -431,3 +431,7 @@ def hashable(v):
     except TypeError:
         return False
     return True
```

Comment:
FWIW all tests pass if this function is replaced with `isinstance(v, Hashable)` (but I don't think worth changing given the risks, even if it seems to work)